### DISCUSSION TITLE
how to use IgnoreUnnamedModuleForSplitPackage

### DISCUSSION THREAD

**Discussion by huige007** [eclipse-jdt/eclipse.jdt.debug#401] [General]
As described in title, how can i enable feature IgnoreUnnamedModuleForSplitPackage. Because I got following error msg in vscode-java(redhat.java)
The package java.util.concurrent is accessible from more than one module: <unnamed>, java.base

after I set the java.jdt.ls.java.home to jdk8, above error msg disappeared

**Comment by iloveeclipse** [comment]
If we would be in Eclipse, adding line below to `<project>/.settings/org.eclipse.jdt.core.prefs`  would do the trick.
No idea what need to be done in VS Code.
```
org.eclipse.jdt.core.compiler.ignoreUnnamedModuleForSplitPackage=enabled
```
